### ISSUE TITLE
CallWithTimeout broken on Windows

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#765]
This is repeatedly happening in the master branch on Windows:

```
testing: /proc/cygdrive/C/gap4r9/tst/testinstall
########> Diff in /proc/cygdrive/C/gap4r9/tst/testinstall/timeout.tst:13
# Input is:
CallWithTimeout(5000,spinFor,50000);
# Expected output:
[ false ]
# But found:
[ true ]
########
```

**Comment by fingolfin** [comment]
See #664 for a suggestion to phase out `CallWithTimeout` again due to major issues with it.

**Comment by fingolfin** [comment]
Resolved by #1324.
